### PR TITLE
CHAMBER: cga engine modified to run hercueles instead of cga.

### PR DESCRIPTION
**Pull request by yigithanyigit**
As you mentioned I changed constants. But there is a problem after I ran that we talked earlier. I guess I am makin wrong memory calculations at some point but I could not understand.

image

### CODE REVIEW

**Comment by yigithanyigit** [comment]
@sev-

**Comment by yigithanyigit** [comment]
@sev-

**Comment by bluegr** [comment]
@yigithanyigit thanks for your work. Please, use our commit guidelines - in particular, your commit should have a prefix of the area it affects. In this case, it's the CHAMBER engine

File: engines/engine.h [hunk 1]
```diff
@@ -235,6 +235,10 @@ class Engine {
 	GUI::Debugger *_debugger;
 public:
 
+	enum VideoMode {
+		kCGAMode,
+		kHercMode,
+	};
```
Comment (bluegr) [inline engines/engine.h:241]: We already have an enum for these, RenderMode. Check [common/rendermode.h](https://github.com/scummvm/scummvm/blob/master/common/rendermode.h)

**Comment by sev-** [comment]
Closing this as unfinished

**Pull request closed without merging**